components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/UserSessionManagementServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+/*
+ *   Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *   WSO2 Inc. licenses this file to you under the Apache License,
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.wso2.carbon.identity.application.authentication.framework;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.application.authentication.framework.exception.UserSessionException;
+import org.wso2.carbon.identity.application.authentication.framework.services.SessionManagementService;
+import org.wso2.carbon.identity.application.authentication.framework.store.UserSessionStore;
+import org.wso2.carbon.identity.application.common.model.User;
+import org.wso2.carbon.identity.core.util.IdentityTenantUtil;
+import org.wso2.carbon.utils.multitenancy.MultitenantUtils;
+
+import java.util.List;
+
+/**
+ * This a service class used to manage user sessions.
+ */
+public class UserSessionManagementServiceImpl implements UserSessionManagementService {
+
+    private static final Log log = LogFactory.getLog(UserSessionManagementServiceImpl.class);
+    private SessionManagementService sessionManagementService = new SessionManagementService();
+
+    @Override
+    public void terminateSessionsOfUser(String userName)
+            throws UserSessionException {
+
+        List<String> sessionIdList;
+        String userId;
+        if (log.isDebugEnabled()) {
+            log.debug("Get session details of user: " + userName);
```

Comment:
What about using "Retrieving session details of user:" ?
Because, at the time your log is printing the session details are not yet retrieved.